src/public/locales/fr/translation_fr.json

```diff
@@ -12,5 +12,105 @@
   "Experiment Types": "Types d'expériences",
   "Types of experiments performed on a sample": "Types d'expériences réalisées sur un échantillon",
   "I have no problems in walking about": "Je n’ai aucun problème à marcher",
-  "Results": "Résultats"
+  "Results": "Résultats",
+  "General": "Général",
+  "Measurements": "Mesures",
+  "Medical Actions": "Actions médicales",
+  "Interpretations": "Interprétations",
+  "Experiments": "Expériences",
+  "Date of initial verbal consent (participant, legal representative or tutor), yyyy-mm-dd": "Date du consentement verbal initial (participant, représentant légal ou tuteur), aaaa-mm-jj",
+  "Covid severity": "Gravité de la Covid",
+  "Dead": "Décédé",
+  "Moderate": "Modéré",
+  "Uninfected": "Non infecté",
+  "Severe": "Sévère",
+  "Mild": "Léger",
+  "Mobility": "Mobilité",
+  "I have slight problems in walking about": "J’ai quelques problèmes à marcher",
```

Comment:
for the terms that come from BQC19, use the BQC19 translations instead of machine translation.